types/index.ts

```diff
@@ -59,6 +67,7 @@ export type ButtonProps = {
   outerContainerStyle?: object;
   buttonStyle?: object;
   textStyle?: object;
+  containerStyle?: object;
```

Comment:
Should we use the buttonStyle?

Reply:
it's already have a buttonStyle. I name it continerStyle because the default style name it as container (that style cover the pressable but the button is inside it)

Reply:
ok